src/components/List/List.js

```diff
@@ -34,6 +34,16 @@ const List = ({ token, setToken }) => {
     return !loading && !error && groceryItems && groceryItems.length;
   };
 
+  const sortByHowSoonAndName = (list) => {
```

Comment:
Love how you created this function to be used in both cases below 😍. Consistency for the win!